packages/core/admin/ee/server/services/seat-enforcement.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const { take, drop } = require('lodash/fp');
+const { getService } = require('../../../server/utils');
+
+const enableUsersToLicenseLimit = async (numberOfUsersToEnable) => {
+  const data = await getService('user').getDisabledUserList();
+
+  if (!data?.value || data.value.length === 0) {
+    return;
+  }
+
+  const disabledUsers = JSON.parse(data.value);
+
+  const usersToEnable = take(disabledUsers, numberOfUsersToEnable);
+
+  for await (const user of usersToEnable) {
+    try {
+      await strapi.db.query('admin::user').update({
+        where: { id: user.id },
+        data: { isActive: true },
+      });
+    } catch (error) {
+      return;
```

Comment:
Shouldn't an error be thrown if anything happens when updating this? 

Or maybe adding a log, so this error is not silent at least.